bigquery/list_datasets_and_projects.js

```diff
@@ -23,11 +23,16 @@
 
 // [START all]
 // [START auth]
-// By default, gcloud will authenticate using the service account file specified
-// by the GOOGLE_APPLICATION_CREDENTIALS environment variable and use the
-// project specified by the GCLOUD_PROJECT environment variable. See
-// https://googlecloudplatform.github.io/gcloud-node/#/docs/guides/authentication
-var gcloud = require('gcloud');
+// By default, the client will authenticate using the service account file
+// specified by the GOOGLE_APPLICATION_CREDENTIALS environment variable and use
+// the project specified by the GCLOUD_PROJECT environment variable. See
+// https://googlecloudplatform.github.io/gcloud-node/#/docs/google-cloud/latest/guides/authentication
+var BigQuery = require('@google-cloud/bigquery');
+
+var Resource = require('@google-cloud/resource');
+
+// Instantiate a resource client
+var resource = Resource();
```

Comment:
All samples put it at the top, except for those samples that specifically need to instantiate a client with a custom option before making a request, like a `projectId`, as does the datasets sample in this file. Typically users will instantiate clients once at the top of some file, not right before they need to make a request.